packages/shared/src/css/styles.css

```diff
@@ -1,10 +1,8 @@
 :root {
-  --p-transition-duration: 0s;
-  --p-animation-duration__spinner: 0s;
-  --p-animation-duration__banner: 0s;
-  --p-override-popover-animation-duration: 0s;
-  --p-override-toast-skip-timeout: true;
-  --p-override-toast-animation-duration: 0s;
+  --pds-transition-duration: 0s;
+  --pds-animation-duration: 0s;
+  --_pds-spinner-stroke-dasharray: 40;
```

Comment:
Looks super weird and wrong to me 🤷  
Things starting with a `_` are considered internal or non public anywhere else.  
But here these variables are actually temporary and hopefully not part of a prod build. Therefore I would  prefer something with `temporary` or `internal` that speaks for itself or even something like `--test-override-whatever`.